README.md

```diff
@@ -89,6 +91,11 @@ This event is bound by the [Hacktoberfest Code of Conduct](https://do.co/hacktob
         <br>
         <span>Retno Ayu Widyanti</span>
       </td>
+        <td align="center" valign="top">
+        <img width="150" height="150" src="https://avatars0.githubusercontent.com/u/5687681?s=460&v=4">
```

Comment:
For consistency with the other avatars, could we please use this URL for the image?

```suggestion
        <img width="150" height="150" src="https://github.com/lorderikir.png?s=150">
```

Reply:
PS woo how cool are the new GitHub suggestions! 😱 🎉

Reply:
@josh- They're super neat!